samples/OpenThread/README.md

```diff
@@ -0,0 +1,82 @@
+# 🌶️🌶️ - OpenTHread Networking sample pack
+
+Shows how to use OpenThread Networking API.
+
+## Samples
+
+- [🌶️🌶️🌶️ - UPD OpenThread Client using sockets](UdpThreadClient/)
+- [🌶️🌶️🌶️ - UPD OpenThread Server using sockets](UdpThreadServer/)
+
+Shows how to use various APIs related to OpenThread.
+
+> **Note:** This sample is part of a large collection of nanoFramework feature samples.
```

Comment:
you can remove this, see other samples, we've been standardizing the readme and this is gone now as now everyone is using github